src/mapping/grouping/attractor_mapping_featurizing.jl

```diff
@@ -166,6 +166,12 @@ function extract_features_single(mapper, ics; show_progress = true, N = 1000)
     return feature_vector
 end
 
+# Mapper function that works with GroupViaNearestFeature
+function (mapper::AttractorsViaFeaturizing)(u0)
+   f = extract_features_single(mapper,[u0]) 
+   return feature_to_group(f[1],mapper.group_config) 
```

Comment:
```suggestion
   f = extract_features_single(mapper,[u0]) 
   return feature_to_group(f[1],mapper.group_config) 
```

Here careful with styling, add space between arguments.